client/src/components/feeds/post/reactions/ReactionBar.tsx

```diff
@@ -234,20 +237,25 @@ function ReactionBar(props: ReactionBarProps) {
         if (!showReactText) {
           showAll();
         }
+        if (hideButtonsTimeout) clearTimeout(hideButtonsTimeout);
       }}
       onMouseLeave={() => {
-        if (reactionCounts.every((count) => count === 0)) {
-          setShowReactText(true);
-        }
-        hideAll();
+        setHideButtonsTimeout(
+          setTimeout(() => {
+            if (reactionCounts.every((count) => count === 0)) {
+              setShowReactText(true);
+            }
+            hideAll();
+          }, 250)
+        );
       }}
     >
       <LoginPopup showPopup={showPopup} closePopup={closePopup} />
       {showReactText && (
         <p
           className="ReactText"
           onClick={
-            props.user
+            props.user || true
```

Comment:
I assume this was added for debugging and wasn't intended to be included in the commit? If it was intended, why is this necessary?

Reply:
Yep, forgot to skip that when committing